lib/msal-node/src/request/ClientCredentialRequest.ts

```diff
@@ -3,15 +3,18 @@
  * Licensed under the MIT License.
  */
 
-import { CommonClientCredentialRequest } from "@azure/msal-common";
+import {
+    ClientAssertionCallback,
+    CommonClientCredentialRequest,
+} from "@azure/msal-common";
 
 /**
  * CommonClientCredentialRequest
  * - scopes                  - Array of scopes the application is requesting access to. Typically contains only the .default scope for a single resource. See: https://learn.microsoft.com/azure/active-directory/develop/scopes-oidc#the-default-scope
  * - authority               - URL of the authority, the security token service (STS) from which MSAL will acquire tokens.
  * - correlationId           - Unique GUID set per request to trace a request end-to-end for telemetry purposes.
  * - skipCache               - Skip token cache lookup and force request to authority to get a a new token. Defaults to false.
- * - clientAssertion         - A Base64Url-encoded signed JWT assertion string used in the Client Credential flow
+ * - clientAssertion         - A callback function that returns a Base64Url-encoded signed JWT assertion string used in the Client Credential flow
```

Comment:
```suggestion
 * - clientAssertionCallback         - A callback function that returns a Base64Url-encoded signed JWT assertion string used in the Client Credential flow
```

Reply:
either the name of the variable is clientAssertionCallback or the description should be string or callback function?

Reply:
the latter